### PR TITLE
Removed references to s2i builds and included cordova install

### PR DESCRIPTION
**Pull request by luigizuccarelli**
@wei-lee @aliok - Please review, as mentioned I have managed to get the centos images to build Also included cordova). The rhel7 version is giving me errors wrt rh-nodejs4 (not found). I will leave this out for now I have asked Sushma and Jozef for help, so this (the rhel7 version is still a WIP)

### CODE REVIEW

File: docker/android-slave/Dockerfile [hunk 1]
```diff
@@ -22,37 +25,35 @@ RUN yum install -y centos-release-scl-rh && \
...
-
 # This default user is created in the openshift/base-centos7 image
 USER 1001
 WORKDIR /opt/workspace
-CMD ["usage"]
 
-ENV ANDROID_HOME=/opt/android-sdk-linux
-ENV PATH=$PATH:$ANDROID_HOME/tools:$ANDROID_HOME/platform-tools
+# TODO: Need to verify this
+#ENV ANDROID_HOME=/opt/android-sdk-linux
```
Comment (wei-lee) [inline docker/android-slave/Dockerfile]: This will be injected by the kubernetes Jenkins plugin

**Comment by wei-lee** [comment]
👍

**Comment by aliok** [comment]
Code looks good to me, but I haven't tried connecting it with Jenkins master yet.

**Comment by luigizuccarelli** [comment]
@wei-lee @aliok - Thanks for the review. If you could verify it on your local vm and let me know so I can make the necessary changes if needed

**Comment by aliok** [comment]
@luigizuccarelli although I am not sure if we need to test as following for sure but I had a problem. 
When I tried using this new image in Jenkins for Android agents, I got this error in the pods:
```
/usr/local/bin/run-jnlp.sh: line 9: exec: c2685844688cd5f56d789668fb3eeb6ef96bc49cb356fff8eee475ea6c034ccd: not found
```
Then the pod dies.

**Comment by luigizuccarelli** [comment]
@aliok - OK I'll look into that now

File: docker/README.md [hunk 2]
```diff
@@ -65,38 +65,41 @@ Jenkins Kubernetes plugin slaves:
...
-This will build the final image with the android sdk version that was installed
+    docker build -t docker.io/aerogear/jenkins-android-slave:2.0.0 
+
+If changes were made then execute the following
+
+    docker push docker.io/aerogear/jenkins-android-slave:2.0.0
+
+The android-sdk will be installed and mounted on a PV (persistent volume in OpenShift) refer to [android-sdk](./android-sdk) 
+
+For configuration of this image :-
```
Comment (PhilipGough) [inline docker/README.md]: Maybe just a note here that this would require the Jenkins Kubernetes Plugin

**Comment by aidenkeating** [comment]
@luigizuccarelli I was able to build an Android Cordova project using this slave in Jenkins, tested it and it worked fine.

I did have to set the Kubernetes Plugin option `Disable https certificate check` locally, otherwise slaves would fail to run due to handshake issues though. So maybe that could go into the README?

**Comment by odra** [comment]
closing it due to #37

**Pull request closed without merging**